### PR TITLE
All repos stars

### PR DESCRIPTION
**Pull request by valdirjunior011**
## Description

Get all stars, forks, subscribers for all repos by owner repository## Type of change

- [x] New script command

## Screenshot

Repository Statistics

## Dependencies / Requirements

Access token
Owner Repository name

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/developer-utils/github/github-repository-stars_all_repos_template.rb [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+# @raycast.refreshTime 1h
+
+# Optional parameters:
+# @raycast.packageName GitHub
+# @raycast.icon ⭐️
+
+# Documentation:
+# @raycast.description Show statistics of your GitHub repositories.
+# @raycast.author Thomas Paul Mann
+# @raycast.authorURL https://github.com/thomaspaulmann
```
Comment (dehesa) [inline commands/developer-utils/github/github-repository-stars_all_repos_template.rb:22]: Don't you want to add your name an github URL instead?

**Review by dehesa — APPROVED** [review]
Looks good!

File: commands/developer-utils/github/github-repository-stars_all_repos_template.rb (hunk 1 above)
Comment (valdirjunior011) [reply commands/developer-utils/github/github-repository-stars_all_repos_template.rb:22]: nah this ok!

**Review by valdirjunior011 — COMMENTED** [review]
done

**Comment by unnamedd** [comment]
@valdirjunior011 sorry to arrive here later, but you cannot change the original author of a Script Command. The author is Thomas Paul Mann, you are a contributor to this Script Command, you can duplicate the metatags `@raycast.author` and `@raycast.authorURL` to add yours, not replace the original author.

Would you kindly revert and re-add the original author's name and add yours as a contributor, please?
cc @dehesa

**Comment by dehesa** [comment]
Hey @unnamedd,

I was the one suggesting him to be the author since this is a new script, albeit very similar to the other one. You can see the [previous conversation here](https://github.com/raycast/script-commands/pull/859).

**Comment by valdirjunior011** [comment]
Sure, Don't problem at all. I just make an new PR adding original Author.

**Comment by dehesa** [comment]
No no, @valdirjunior011. It is alright. This has been a bit of a misunderstanding. Sorry! Thank you again for the contribution and I will talk with @unnamedd privately to explain.

**Comment by unnamedd** [comment]
@valdirjunior011 everything is clear! 
Thanks for your contribution and please, keep 'em coming!

**Pull request merged**